app/views/buildings/_summary_table.html.erb

```diff
@@ -0,0 +1,26 @@
+<h3><%= building.name %></h3>
+<table>
+  <thead>
+    <tr>
+      <td>Suite Size</td>
+      <td># Suites</td>
+      <td># Suites Assigned</td>
+      <td># Suites in Draws</td>
+      <td># Suites not in a Draw</td>
+    </tr>
+  </thead>
+  <tbody>
+    <% @suites_by_size[building.id].each_value do |size, suites| %>
+      <tr>
+        <td data-role="suite-size"><%= size_str(size).capitalize.pluralize %></td>
+        <td data-role="suite-count"><%= suites.count %></td>
+        <td data-role="assigned-count"><%= suites.where.not(group_id: nil).count %></td>
```

Comment:
Suites is an array, not an association (because of the `group_by` call in the association) so we need to use Ruby Enumerable methods.